src/procedures/procedures.11tydata.yaml

```diff
@@ -308,3 +308,21 @@ procedureReassembleCase:
           <li>Insert the 4 thumb screws.</li>
           <li class="warning">Do not over-tighten the screws.</li>
         </ul>
+procedureRemoveCM4:
+  title: Removing the CM4
+  steps:
+    - title: Removing the heat sink
+      image: remove-pins-03.webp
+      content: |
+        <ul>
+          <li>Use the pliers and squeeze the front of the pin so that it fits through the hole.</li>
+          <li>Repeat the procedure for the other pin.</li>
+        </ul>
+    - title: Removing the Raspberry Pi Compute Module 4
+      image: unseat-cm4.webp
+      content: |
+        <ul>
+          <li>Pull up the module from one side.</li>
```

Comment:
Add one note something along the lines: "Ideally the CM4 should be pulled off vertically, but that is hard to do. However, try to tilt the CM4 as little as possible to avoid damage to the connector."

Reply:
thank you @agners. Implemented.